src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1151,6 +1237,106 @@ private static boolean likelihoodsAreUninformative(final double[] likelihoods) {
         return alleleIndexesToKeep;
     }
 
+    /**
+     * Find the strand allele count indexes to use for a selected set of alleles
+     *
+     * @param originalVC   the original VariantContext
+     * @param allelesToUse the subset of alleles to use
+     * @return a list of SAC indexes to use or null if none
+     */
+    public static List<Integer> determineSACIndexesToUse(final VariantContext originalVC, final List<Allele> allelesToUse) {
+
+        Utils.nonNull(originalVC, "VariantContext is null");
+        Utils.nonNull(allelesToUse, "Alleles to use is null");
+
+        // the bitset representing the allele indexes we want to keep
+        final boolean[] alleleIndexesToUse = getAlleleIndexBitset(originalVC, allelesToUse);
+
+        // an optimization: if we are supposed to use all (or none in the case of a ref call) of the alleles,
+        // then we can keep the SACs as is; otherwise, we determine which ones to keep
+        if (MathUtils.countOccurrences(true, alleleIndexesToUse) == alleleIndexesToUse.length)
```

Comment:
Done. Changed logic to `if ( alleleIndexesToUse.cardinality() == alleleIndexesToUse.size() )` since `alleleIndexesToUse` is now a `BitSet`.